application/controllers/TemplatesController.php

```diff
@@ -23,6 +23,7 @@ class TemplatesController extends Controller
     public function indexAction()
     {
         $this->createTabs()->activate('templates');
+        $this->getTabs()->getAttributes()->set('data-base-target', '_main');
```

Comment:
I wonder why that's not in the trait already? Please add it there

Reply:
Which trait?

Reply:
`ReportsTimeframesAndTemplatesTabs`